Client/style/style.css

```diff
@@ -0,0 +1,3013 @@
+/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```

Comment:
Add this to .gitignore and let each client compile it.

NOTE: This may have issues on deployment if we cannot complie SCSS on Azure